### PR TITLE
README.md multimodal params fix

### PR DESCRIPTION
**Pull request by zocainViken**
multi modal params fix: add logits = True -> to make llava work

### CODE REVIEW

**Comment by abetlen** [comment]
@zocainViken thank you!
